README.md

```diff
@@ -1,3 +1,34 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Submariner](#submariner)
+- [Architecture](#architecture)
+      - [submariner](#submariner)
+      - [submariner-route-agent](#submariner-route-agent)
+    - [Network Path](#network-path)
+- [Prerequisites](#prerequisites)
+- [Installation](#installation)
+  - [Manual installation using helm charts](#manual-installation-using-helm-charts)
+    - [Setup](#setup)
+    - [Broker Installation/Setup](#broker-installationsetup)
+    - [Submariner Installation/Setup](#submariner-installationsetup)
+      - [Installation of Submariner in each cluster](#installation-of-submariner-in-each-cluster)
+  - [Installation using operator](#installation-using-operator)
+    - [Generating the Operator](#generating-the-operator)
+    - [Builiding the operator](#builiding-the-operator)
+    - [Deploying Submariner using the Operator](#deploying-submariner-using-the-operator)
+  - [Validate Submariner is Working](#validate-submariner-is-working)
+- [Testing](#testing)
+- [Known Issues/Notes](#known-issuesnotes)
+    - [AWS Notes](#aws-notes)
+    - [Openshift Notes](#openshift-notes)
+- [Building](#building)
+- [Contributing](#contributing)
+- [TODO](#todo)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
```

Comment:
note: if we will be using doctoc to maintain this I'd add it to the make script / dapper image.

Reply:
^ (this does not require action, I could help with that on a later PR)